package.json

```diff
@@ -2,47 +2,46 @@
   "dependencies": {
     "@react-google-maps/api": "^2.18.1",
     "clsx": "^1.2.1",
-    "next": "13.2.1",
-    "pnp": "^0.0.3",
-    "pnpm": "^7.27.1",
+    "next": "13.4.3",
     "react": "18.2.0",
     "react-dom": "18.2.0"
   },
   "description": "The next version of halfstackconf.com. ✨",
   "devDependencies": {
-    "@axe-core/playwright": "^4.6.0",
-    "@playwright/test": "^1.31.1",
-    "@types/node": "18.14.1",
-    "@types/react": "18.0.28",
-    "@types/react-dom": "18.0.11",
-    "@typescript-eslint/eslint-plugin": "^5.53.0",
-    "@typescript-eslint/parser": "^5.53.0",
-    "eslint": "8.34.0",
-    "eslint-config-next": "13.2.1",
-    "eslint-config-prettier": "^8.6.0",
-    "eslint-plugin-jsonc": "^2.6.0",
+    "@axe-core/playwright": "^4.7.1",
+    "@playwright/test": "^1.34.2",
+    "@types/node": "20.2.3",
+    "@types/react": "18.2.6",
+    "@types/react-dom": "18.2.4",
+    "@typescript-eslint/eslint-plugin": "^5.59.7",
+    "@typescript-eslint/parser": "^5.59.7",
+    "eslint": "8.41.0",
+    "eslint-config-next": "13.4.3",
+    "eslint-config-prettier": "^8.8.0",
+    "eslint-plugin-jsonc": "^2.8.0",
     "eslint-plugin-jsx-a11y": "^6.7.1",
     "eslint-plugin-react": "^7.32.2",
     "eslint-plugin-simple-import-sort": "^10.0.0",
-    "eslint-plugin-typescript-sort-keys": "^2.1.0",
+    "eslint-plugin-typescript-sort-keys": "^2.3.0",
     "husky": "^8.0.3",
-    "lint-staged": "^13.1.2",
-    "markdownlint-cli": "^0.33.0",
+    "lint-staged": "^13.2.2",
+    "markdownlint-cli": "^0.34.0",
     "npm-package-json-lint": "^6.4.0",
     "npm-package-json-lint-config-default": "^5.0.0",
-    "playwright": "^1.31.1",
-    "prettier": "^2.8.4",
-    "stylelint": "^15.2.0",
-    "stylelint-config-standard": "^30.0.1",
-    "stylelint-order": "^6.0.2",
+    "playwright": "^1.34.2",
+    "prettier": "^2.8.8",
+    "stylelint": "^15.6.2",
+    "stylelint-config-standard": "^33.0.0",
+    "stylelint-order": "^6.0.3",
     "ts-prune": "^0.10.3",
-    "typescript": "4.9.5"
+    "typescript": "5.0.4"
   },
   "license": "UNLICENSED",
   "lint-staged": {
     "*": "prettier --ignore-unknown --write"
   },
   "name": "halfstackconf-next",
+  "packageManager": "pnpm@8.5.0",
```

Comment:
This should help inform what package manager to use, though it doesn't enforce much yet. https://nodejs.org/dist/latest-v20.x/docs/api/all.html#all_packages_packagemanager